packages/liveblocks-react/src/types.ts

```diff
@@ -1215,6 +1215,18 @@ type LiveblocksContextBundleCommon = {
    * markAllInboxNotificationsAsRead();
    */
   useMarkAllInboxNotificationsAsRead(): () => void;
+
+  /**
+   * @beta
+   *
+   * Returns the inbox notification thread for the given inbox notification ID.
+   *
+   * @example
+   * const thread = useInboxNotificationThread("in_xxx");
+   */
+  useInboxNotificationThread(
+    inboxNotificationId: string
+  ): ThreadData<BaseMetadata>;
```

Comment:
I forgot about this, but we need to allow proper typing of threads here, and unfortunately `createLiveblocksContext` doesn't have a `TThreadMetadata` generic yet. I can look into this or help if you want as it's a bit all over the place.

Reply:
The pieces were already there in the end so it was easier than I thought, I pushed a commit to use `TThreadMetadata` everywhere.